test-server/test-echo.c

```diff
@@ -33,6 +33,9 @@
 #include <syslog.h>
 #include <sys/time.h>
 #include <unistd.h>
+#else
+#include "../win32port/win32helpers/gettimeofday.h"
```

Comment:
Simply include `"gettimeofday.h"`... The include path is added here: https://github.com/warmcat/libwebsockets/blob/master/CMakeLists.txt#L416